### ISSUE TITLE
Snippets does not work

### ISSUE THREAD

**Issue by Ciantic** [emmetio/emmet-atom#242]
Reproduce:

type `input`, click from menu Packages -> Emmet -> Expand abbreviation, it expands to `<input></input>` likewise with `html:5` or others.

**Comment by Ciantic** [comment]
Apparently only non html files are affected, I use TypeScript and JSX it's pretty important to get the html snippets to work in all files where Emmet works too.

**Comment by sergeche** [comment]
Please check DevTools console for possible errors

**Comment by Ciantic** [comment]
@sergeche there is no errors. Just a warning I don't have `C:/Users/username/emmet` directory. Btw, this error may be duplicate of #104

**Comment by sergeche** [comment]
Can you send me package names you’re using for JSX and TypeScript?

**Comment by Ciantic** [comment]
@sergeche I don't think this has anything to do with TypeScript or JSX. The snippets just doesn't work right in non-html files. such as something.json, something.js or something.ts.

Also I don't have any special jsx package, I use atom-typescript and newly introduced JavaScript feature [templating string](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/template_strings) (backticks) basically a multiline string for JSX.

**Comment by Spentacular** [comment]
I'm getting this same problem as well now.

My console:

```
Loading Emmet extensions from ~/emmet
/Users/spentacular/.atom/packages/emmet/lib/emmet.coffee:150 Emmet: no such extension folder: /Users/spentacular/emmet
```

I currently unable to expand anything besides the basic class or id structure. `a` results in `<a></a>`, etc.

**Comment by igorprado** [comment]
Same problem here.

Using for JSX syntax: https://atom.io/packages/react

Console warnings: 

```
Loading Emmet extensions from ~/emmet
/Users/username/.atom/packages/emmet/lib/emmet.coffee:150 Emmet: no such extension folder: /Users/username/emmet```

Emmet version: 2.3.10
Atom: 0.201.0

**Comment by patrickarlt** [comment]
I am also seeing this same behavior when using non-html grammers. When I am in HTML (Ruby - ERB) mode i can only expand basic htmls taga like `a` or `input` but more complex things like `a.button` fail to expand.

I am also seeing the same errors as other users. If I switch to just the plain HTML grammer everything works fine but then I lose my syntax highlighting for ERB.

```
Loading Emmet extensions from ~/emmet
/Users/patricklocal/.atom/packages/emmet/lib/emmet.coffee:150 Emmet: no such extension folder: /Users/patricklocal/emmet
``

Atom: 0.211.0
Emmet: 2.3.10

**Comment by sergeche** [comment]
> When I am in HTML (Ruby - ERB) mode i can only expand basic htmls taga like a or input but more complex things like a.button fail to expand.

See https://github.com/emmetio/emmet-atom#tab-key

**Comment by sergeche** [comment]
Warnings like
```
Loading Emmet extensions from ~/emmet
/Users/patricklocal/.atom/packages/emmet/lib/emmet.coffee:150 Emmet: no such extension folder: /Users/patricklocal/emmet
```

...simply tells you where Emmet looks for [custom extensions](http://docs.emmet.io/customization/), it doesn’t break anything.

**Comment by Ciantic** [comment]
This problem has nothing to do with tab key, as I demonstrated, I clicked from a menubar: Packages -> Emmet -> Expand abbreviation - not using the shortcut (which btw is ctrl+enter on my mapping at the moment)

Either way, it still expands incorrectly "input" to "<input></input>" in normal text file or anything else than HTML.

**Comment by sergeche** [comment]
Should be fixed in v2.3.11

**Comment by lid3rs** [comment]
Passed two years, same problem.

<img width="369" alt="screen shot 2017-07-21 at 14 05 29" src="https://user-images.githubusercontent.com/1872001/28461090-b99693e8-6e1d-11e7-920d-d2056da58d97.png">

**Comment by sergeche** [comment]
@lid3rs it’s not a problem, it’s just a warning telling that you have no custom Emmet extensions

**Comment by itwebmaster** [comment]
=)) Hi guys.... We can read last paragraph of [readme.md this plugin](https://github.com/emmetio/emmet-atom)
## **Extensions support** 
You can easily extend Emmet with new actions and filters or customize existing ones. In Preferences > Emmet, set Extensions path to folder with Emmet extensions. By default, it’s ~/emmet, e.g. emmet folder in your system HOME folder.

## So...
This plugin try find this folder (~/emmet = <winDrive>:\Users\<CurrentUser>\emmet - in Windows)
You can just create this folder (empty) ;)

**Comment by itwebmaster** [comment]
=)) Hello guys! You can read the last paragraph [readme.md of this plugin](https://github.com/emmetio/emmet-atom#extensions-support):

> ### Support extension
> You can easily extend Emmet with new actions and filters or customize existing ones. In the "Settings"> "Emmet" section, set the extension path to the folder with the Emmet extensions. The default is ~/emmet. emmet in the home directory of your system.
> 

Hence the conclusion
This plugin tries to find this folder "~ /emmet"  _("<WinDrive(C)>:\Users\<CurrentUser>\emmet" - on Windows)_

## And the solution to this problem.

- You just need to create this folder (**empty**) _("<WinDrive(C)>:\Users\<CurrentUser>\emmet" - on Windows)_
_OR_
- [If you want to extend emmet](https://docs.emmet.io/customization/)
